### PR TITLE
 chore: backport upgrade to node 18

### PR DESCRIPTION
**Pull request by regisb**
## Description

Backport the following PR: https://github.com/openedx/frontend-app-payment/pull/723

This backport is necessary to build the microfrontend apps of the ecommerce application in the open-release/palm.master branch with node 18 (and not node 16).

## Supporting information

Issue was raised here: https://github.com/openedx/wg-build-test-release/issues/261#issuecomment-1584746372

## Testing instructions

Check that this version builds with the "palm" branch of the tutor-ecommerce plugin: https://github.com/overhangio/tutor-ecommerce/pull/42

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @regisb!

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.

**Review by pshiu — APPROVED** [review]

**Review by arbrandes — APPROVED** [review]

**Pull request merged**
